client/signup/steps/reader-landing/content.jsx

```diff
@@ -22,8 +22,7 @@ class ReaderLandingStepContent extends PureComponent {
 
 	render() {
 		const { translate } = this.props;
-		return (
-			<Card className="reader-landing__step-content">
+		return (			
 				<div className="reader-landing__button-wrapper">
```

Comment:
Adjacent elements need a wrapper in JSX - you can check out [React fragments](https://reactjs.org/docs/fragments.html). Calypso supports the shorthand.

There also seems to be some additional indentation here. It might be worth checking out the doc on Calypso's [coding guidelines](https://github.com/Automattic/wp-calypso/blob/562814891e58a884a8093c912eedc3f0dcd096b4/docs/coding-guidelines/javascript.md) and setting up [linting for the project](https://github.com/Automattic/wp-calypso/blob/562814891e58a884a8093c912eedc3f0dcd096b4/docs/coding-guidelines/javascript.md#eslint).

Reply:
Sorry for adding a commit and then abandoning this; I couldn't sleep so I looked at this, and as the tests are running, then I feel tired. :roll_eyes: 

Thank you very much for the helpful links and review though! 

> Adjacent elements need a wrapper in JSX - you can check out React fragments. Calypso supports the shorthand

So I understand this correctly here, I'm assuming a `<>` on line 26 and 126 will solve this issue? GitHub isn't being very helpful in searching that phrase, so I can't check for examples on if Calypso wraps a div around it or just uses `<>`. (I added a commit for now, based on your review, I'm assuming it's okay to go with `<>`). 

> There also seems to be some additional indentation here.

I'm assuming the indententation will be okay with the above added.

> setting up linting for the project.

I'm not sure why this wasn't working for my second commit, but it looks like the tests now pass and the linter is happy!